cmd/spicedb/serve_integration_test.go

```diff
@@ -0,0 +1,82 @@
+//go:build docker
+// +build docker
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+	"github.com/authzed/grpcutil"
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+	"google.golang.org/grpc/status"
+)
+
+func TestServe(t *testing.T) {
+	requireParent := require.New(t)
+
+	tester, err := newTester(t,
+		&dockertest.RunOptions{
+			Repository:   "authzed/spicedb",
+			Tag:          "currentpr",
+			Cmd:          []string{"serve", "--log-level", "debug", "--grpc-preshared-key", "firstkey", "--grpc-preshared-key", "secondkey"},
+			ExposedPorts: []string{"50051/tcp"},
+		},
+		"firstkey",
+	)
+	requireParent.NoError(err)
+	defer tester.cleanup()
+
+	for key, expectedWorks := range map[string]bool{
+		"":           false,
+		"firstkey":   true,
+		"secondkey":  true,
+		"anotherkey": false,
+	} {
+		t.Run(key, func(t *testing.T) {
+			require := require.New(t)
```

Comment:
Not required, but fwiw I prefer to just use require directly and not shadow the import.

Reply:
I do too, but I was keeping to the style in the existing integration tests